### PR TITLE
feat(LINGUIST-315): Add MCQ Test, Question Endpoints, Increasing and Decreasing Confidence

### PR DESCRIPTION
**Pull request by tolgaozgun**
Add MCQ Test, Question Endpoints, Increasing and Decreasing Confidence

### CODE REVIEW

**Review by can-aslan — DISMISSED** [review]
LGTM

File: mcq/views/finish_mcq_test.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        return generate_error_response(400, "All questions must be answered before completing the test")
+
+    questions: list[MCTQuestion] = test.questions.all()
+
+    # Update UnknownWord confidence levels based on answers
+    for question in questions:
+        if question.isUserCorrect:
+            UnknownWord.objects.filter(word=question.word, email=email).first().increase_confidence(INCREASE_CONFIDENCE_ON_CORRECT_MCQ_ANSWER)
+        else:
+            UnknownWord.objects.filter(word=question.word, email=email).first().decrease_confidence(INCREASE_CONFIDENCE_ON_CORRECT_MCQ_ANSWER)
```
Comment (KardelenCeren) [inline mcq/views/finish_mcq_test.py]: The constant should be for decreasing confidence

**Review by KardelenCeren — CHANGES_REQUESTED** [review]
Looks very good! Two things: 

- Can you add swagger info when you get the chance?
- How do we let the MySQL database know that the confidence levels are updated, couldn't figure it out

**Review by KardelenCeren — APPROVED** [review]
LGTM, thanks!

**Pull request merged**